docs/reference/ml/trained-models/apis/put-trained-models.asciidoc

```diff
@@ -684,6 +687,110 @@ include::{es-repo-dir}/ml/ml-shared.asciidoc[tag=inference-config-nlp-tokenizati
 ======
 =====
 
+`question_answering`:::
+(Optional, object)
+include::{es-repo-dir}/ml/ml-shared.asciidoc[tag=inference-config-nlp-question-answering]
++
+.Properties of question_answering inference
+[%collapsible%open]
+=====
+`max_answer_length`::::
+(Optional, integer)
+The maximum amount of words that the answer consists. Defaults to `15`.
+
+`tokenization`::::
+(Optional, object)
+include::{es-repo-dir}/ml/ml-shared.asciidoc[tag=inference-config-nlp-tokenization]
+Recommended to set `max_sentence_length` to `386` with `128` of `span` and set 
+`truncate` to `none`.
++
```

Comment:
> +

This is currently not displaying properly. If my other suggestion doesn't solve this problem, we might need to put this whole tokenization description within a "--" block. Lmk if you want me to help play